services/src/main/java/com/linkedin/kmf/services/configs/ConsumeServiceConfig.java

```diff
@@ -42,6 +42,10 @@
   public static final String CONSUMER_PROPS_CONFIG = "consume.consumer.props";
   public static final String CONSUMER_PROPS_DOC = "The properties used to config consumer in consume service.";
 
+  public static final String LATENCY_SLA_MS_CONFIG = "consume.latency.sla.ms";
```

Comment:
Yes, it is sort of like that. The user can pass in a map which may potentially contain configuration of the same name in difference services. It is useful to put service name in the prefix of config name to distinguish between configs of difference services. See https://github.com/linkedin/kafka-monitor/blob/master/config/kafka-monitor.properties for example config file. In this example, `zookeeper.connect ` is a config used by `BasicEndToEndTest ` while `produce.producer.props ` is a config used by `ProduceService` started as part of `BasicEndToEndTest`.